opensearch.tf

```diff
@@ -0,0 +1,179 @@
+locals {
+  cluster_name   = "opensearch-${terraform.workspace}"
+  cluster_domain = "aepps.com"
+  es_linked_role = data.aws_iam_role.service_linked_role.id
+}
+
+data "aws_region" "current" {}
+
+data "aws_iam_role" "service_linked_role" {
+  name = "AWSServiceRoleForAmazonElasticsearchService"
+}
+
+provider "elasticsearch" {
+  url                   = "https://${local.cluster_name}.${local.cluster_domain}"
+  aws_region            = data.aws_region.current.name
+  elasticsearch_version = "7.10.2"
+  healthcheck           = false
+}
+
+module "opensearch" {
+  source          = "./terraform-aws-opensearch"
+  cluster_name    = local.cluster_name
+  cluster_domain  = local.cluster_domain
+  cluster_version = "1.0"
+
+  warm_instance_enabled   = var.warm_instance_enabled[terraform.workspace]
+  master_instance_count   = var.master_instance_count[terraform.workspace]
+  master_instance_enabled = var.master_instance_enabled[terraform.workspace]
+  hot_instance_count      = var.hot_instance_count[terraform.workspace]
+  availability_zones      = var.availability_zones[terraform.workspace]
+  master_user_name        = "es-admin"
+  master_user_password    = var.opensearch_master_user_password[terraform.workspace]
+  ebs_enabled             = var.ebs_enabled[terraform.workspace]
+  volume_size             = var.volume_size[terraform.workspace]
+  hot_instance_type       = var.hot_instance_type[terraform.workspace]
+  aws_iam_service_linked_role_es = local.es_linked_role
+}
+
+resource "null_resource" "es_backend_role_cluster" {
```

Comment:
Can we handle access to OS with standard AWS policies and service accounts ?

Reply:
Nope , you need saml auth for that. We spoke about it.

Reply:
I remember, but I'm not convinced I'll double check first.